clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonLoadBalancerProvider.groovy

```diff
@@ -67,7 +67,7 @@ class AmazonLoadBalancerProvider implements LoadBalancerProvider<AmazonLoadBalan
 
   private static boolean applicationMatcher(String key, String applicationName) {
     // aws:loadBalancers:account:region:loadbalancer-name:vpc:type
-    return key ==~ 'aws:[^:]*:[^:]*:[^:]*:' + applicationName + '(-[^:]*):'
+    return key ==~ 'aws:[^:]*:[^:]*:[^:]*:' + applicationName + '(-[^:]*)?:.*'
```

Comment:
Don't want to get in the way of this bug fix _but_ it'd be nice to have some unit tests here.

Reply:
good point, will add some!